devimint/src/external.rs

```diff
@@ -130,7 +131,9 @@ impl Bitcoind {
         if !skip_setup {
             // mine blocks
             let blocks = 101;
-            let address = block_in_place(|| client.get_new_address(None, None))?.assume_checked();
+            let address = block_in_place(|| client.get_new_address(None, None))?
+                .require_network(Network::Bitcoin)
```

Comment:
This will break all of our test environments. `Network::Bitcoin` is for mainnet, we use regtest locally to test and also use signet/mutinynet.

I don't think this is a great first issue for a new contributor to tackle, it requires knowing some nuances about the wallet module and when an address has been checked and is safe to use. We can't just blindly assume everyone is using mainnet.

I would encourage you to look for issues tagged with `good first issue` or join the dev call on Mondays to figure out a good issue to get started with.

Reply:
Interesting. 
If we're assuming that not everyone is using mainnet, then there needs to be a way to configure the network during system startup.
 Point me to any details related to the wallet module; I’ll take it from there and continue working on this issue. 

I'll also join the dev call on Monday.